venia-integration-tests/src/tests/pageBuilder/homePage.spec.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
```

Comment:
Is this needed?

Reply:
Yeah, it is needed for IntelliSense to suggest cypress auto completions.